spaceship/spec/provisioning_profile_spec.rb

```diff
@@ -54,6 +54,30 @@
     end
   end
 
+  describe '#all via xcode api' do
+    around(:all) do |example|
+      switch = ENV['SPACESHIP_AVOID_XCODE_API']
+      example.run
```

Comment:
I mean to ensure we set back the `ENV[]` to its original state. Otherwise it breaks the next test as well.